app/js/controller/list_controller.js

```diff
@@ -73,6 +73,9 @@ export default class ListController extends Controller {
       for (let manifestURL in this.list) {
```

Comment:
A bit out of the scope of this bug: instead of using the 'visibilitychange' event to detect app installs/uninstalls, you can hook the ```navigator.mozApps.mgmt.oninstall``` and ```navigator.mozApps.mgmt.onuninstall```. See here:
https://developer.mozilla.org/en-US/docs/Web/API/DOMApplicationsManager
http://mxr.mozilla.org/mozilla-central/source/dom/webidl/Apps.webidl#127